src/main/java/org/jabref/gui/frame/UiMessageHandler.java

```diff
@@ -0,0 +1,10 @@
+package org.jabref.gui.frame;
+
+import java.util.List;
+
+import org.jabref.logic.UiCommand;
+
+public interface UiMessageHandler {
```

Comment:
Please add a little bit of JavaDoc explaining the intention of this interface.

Reply:
done